portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const URL = "https://teachablemachine.withgoogle.com/models/8A06Q2Ycp/";
+
+let model, webcam, labelContainer, maxPredictions;
+
+// load image model, set up webcam
+async function predictImage() {
+  const modelURL = URL + "model.json";
+  const metadataURL = URL + "metadata.json";
+
+  // load model, metadata
+  model = await tmImage.load(modelURL, metadataURL);
+  // Model can predict what is seen by webcam
+  // either as Squidward or Spongebob
+  maxPredictions = model.getTotalClasses();
+
+  // set up webcam
+  const flip = true; // whether to flip the webcam
+  webcam = new tmImage.Webcam(200, 200, flip);
+  await webcam.setup(); // request access to the webcam
+  await webcam.play();
+  window.requestAnimationFrame(loop);
+
+  // append webcam and prediction to DOM
+  document.getElementById("webcam-container").appendChild(webcam.canvas);
+  labelContainer = document.getElementById("image-prediction");
+  for (let i = 0; i < maxPredictions; i++) {
+    labelContainer.appendChild(document.createElement("div"));
+  }
```

Comment:
jason did something similar, and i asked him if hhe could merge this for loop into the for loop in predict() since they're both iterating over maxPredictions and updating the labelContainer. can you do the same? check with him or ping me if you are confused